content/en/Community/_index.md

```diff
@@ -50,4 +50,118 @@ Kyverno is currently a CNCF sandbox project is growing fast. To qualify for the
 
 The goal of this adopters program is to gather real-world usage examples and help Kyverno achieve the incubating state criteria. We hope to achieve this by learning from the existing and new adopters of Kyverno who are willing to share their stories with CNCF. The adopters can remain private or choose to be listed in the public [Adopters List](https://github.com/kyverno/kyverno/blob/main/ADOPTERS.md).
 
-To participate, kindly fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLSeMJxbK6eQ-MpsbaXZ3GKp8NEz9F932keL2pwznI1JIfi7Iqw/viewform) (It will take 5 mins of your time).
\ No newline at end of file
+To participate, kindly fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLSeMJxbK6eQ-MpsbaXZ3GKp8NEz9F932keL2pwznI1JIfi7Iqw/viewform) (It will take 5 mins of your time).
+
+
+### Project Governance
+
+This document identifies the role and responsibilities for the Kyverno community members. It also highlights the requirements for anyone who is looking to take on leadership roles in the Kyverno project.
+
+In order to perform any actions on the Kyverno Github repositories, contributors should have certain access to GitHub, such as creating a pull request in a repository or changing an organization's billing settings, a contributor must have sufficient access to the relevant account or resource.
+
+**Note:** Please make sure to read and observe our [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+### Leadership Roles
```

Comment:
Would suggest calling this `Project Roles` instead of `Leadership Roles` as we want to describe different levels of contribution and engagement in the community.